config/default.yml

```diff
@@ -135,6 +135,13 @@ Sorbet/ForbidRBIOutsideOfAllowedPaths:
   Include:
   - "**/*.rbi"
 
+Sorbet/ForbidRBSAbstract:
+  Description: 'Forbids usage of the `@abstract` annotation with RBS signatures.'
```

Comment:
Can we clarify that this is `@abstract` annotations on _methods_ and not on _classes_ or _modules_? I think we can add that to both the cop name and description.